### DISCUSSION TITLE
Objects added in versions 3+

### DISCUSSION THREAD

**Discussion by Alex-iiii** [Keriew/augustus#762] [Q&A]
Hello!
Is it possible to make some new objects added in versions 3+ appear in game version   2.0.1?
I would like to see decoratve stuff (e.g, garden walls), but  not other Additions. 
Can  I do this by replacing some file or else?

**Comment by crudelios** [comment]
Unfortunately this is not easily possible, the new objects are integrated to the code for the newest versions.

The only way to do that was to grab the needed code of the newest version and add it to the older ones.

**Reply by Alex-iiii** [reply]
> Unfortunately this is not easily possible, the new objects are integrated to the code for the newest versions.
> 
> The only way to do that was to grab the needed code of the newest version and add it to the older ones.

Thanks for reply!
Are they  "integrated to the code" in a single file?
I'll try to edit these parts  - so what file should I  look for?

**Reply by crudelios** [reply]
No, it's actually divided in a lot of files.

There's code for its placement, other code for its effects, yet more code for its graphics, all in different places.
